lib/experimental-default-theme.json

```diff
@@ -131,6 +131,42 @@
 						"slug": "huge",
 						"value": 48
 					}
+				],
+				"fontFamilies": [
+					{
+						"value": "-apple-system,BlinkMacSystemFont,\"Segoe UI\",Roboto,Oxygen-Sans,Ubuntu,Cantarell,\"Helvetica Neue\",sans-serif",
+						"slug": "system",
+						"name": "System",
+						"group": "web-safe"
```

Comment:
What do we need `group` for?

Reply:
As @aristath referred is to allow fonts to be grouped together eg: web-safe fonts, google fonts, theme x fonts etc.

Reply:
Looking at how themes use font-families they tend to work with a handful of them, so I'd think grouping them by providers/categories would be unnecessary.

A different (and more interesting) use case would be to look at how themes/plugins can provide its own UI controls: some of them may choose to add hundreds of fonts so grouping may make sense there, other ones may want to replace the "font family" control for a combination of base scale and ratio, etc.